.github/dependabot.yml

```diff
@@ -1,5 +1,4 @@
 ---
-
 # Any ignore directives should be uncommented in downstream projects to disable
```

Comment:
I'm not sure why this line was removed.

```suggestion

# Any ignore directives should be uncommented in downstream projects to disable
```

Reply:
Me neither - I must've fat-fingered that somehow and not noticed it.  Thanks for catching it!